### PR TITLE
Mobile responsive

### PR DESCRIPTION
**Pull request by kagz**

### CODE REVIEW

**Review by SevenCatsHere — CHANGES_REQUESTED** [review]
Too much additional work on this branch

**Review by mpalmr — CHANGES_REQUESTED** [review]
Shouldn't have all this additional stuff in it.

File: src/components/index.jsx [hunk 1]
```diff
@@ -84,7 +92,7 @@ function Home() {
 							{/* card smaple */}
 							{
 								jobs.map(job => (
-									<div className="col-lg-4" key={job.id}>
+									<div className="col-lg-4 col-md-6 col-xs-12 " key={job.id}>
```
Comment (mpalmr) [inline src/components/index.jsx:94]: Use `Col`  component from `react-bootstrap`

File: src/components/index.jsx [hunk 2]
```diff
@@ -38,13 +51,8 @@ function Home() {
 				<div className="main-body">
 					<h2>Search For Developer Jobs</h2>
 					<div className="main-card">
-						<div>
+						<div >
```
Comment (mpalmr) [inline src/components/index.jsx]: What's the need for this `div`?

File: src/components/index.jsx [hunk 3]
```diff
@@ -38,13 +51,8 @@ function Home() {
...
 							<Form className="search-part">
```
Comment (mpalmr) [inline src/components/index.jsx]: I'd call this `search-form`

File: src/components/index.jsx [hunk 4]
```diff
@@ -38,13 +51,8 @@ function Home() {
 				<div className="main-body">
 					<h2>Search For Developer Jobs</h2>
 					<div className="main-card">
```
Comment (mpalmr) [inline src/components/index.jsx:53]: This isn't really a card, I'd in fact use `Container` from `react-bootstrap`

File: src/components/index.jsx (hunk 1 above)
Comment (mpalmr) [reply src/components/index.jsx:94]: Should also be encased inside of a `<Row>` for all the columns.

File: src/styles/index.scss [hunk 5]
```diff
@@ -173,6 +167,36 @@ font-size: 20px;
...
+	background: #097205;
+}
+
+// Small devices (landscape phones, 576px and up)
+@media (min-width: 576px) {
+}
+
+// Medium devices (tablets, 768px and up)
+@media (min-width: 768px) {
+	.search-part {
```
Comment (mpalmr) [inline src/styles/index.scss]: I'd put these `@media` queries **inside** of the element they're being applied to:
```scss
.search-form {
  foo: bar;
  @media (min-width: 768px) {
    foo: baz;
  }
}

File: src/styles/index.scss [hunk 6]
```diff
@@ -173,6 +167,36 @@ font-size: 20px;
...
+// Large devices (desktops, 992px and up)
+@media (min-width: 992px) {
+	.search-part {
+		display: flex;
+		flex-direction: row;
+	}
+}
+
+// Extra large devices (large desktops, 1200px and up)
+@media (min-width: 1200px) {
```
Comment (mpalmr) [inline src/styles/index.scss]: Remove

File: src/styles/index.scss [hunk 7]
```diff
@@ -173,6 +167,36 @@ font-size: 20px;
 	letter-spacing: 0.2px;
 	font-size: 17px;
 }
-.view-btn:hover{
-	background: #097205;;
+.view-btn:hover {
```
Comment (mpalmr) [inline src/styles/index.scss]:
```scss
.view-btn {
  blah: blah;
  &:hover {
    blah: blah;
  }
}
```

File: src/styles/index.scss [hunk 8]
```diff
@@ -114,56 +111,53 @@ margin: 0;
...
+	padding: 0;
 }
 
-.job-description ul li{
+.job-description ul li {
 	display: flex;
 	// flex-direction: column;
 	// align-items: center;
 	margin: 6px;
 }
```
Comment (mpalmr) [inline src/styles/index.scss:158]:
```scss
.job-description {
  foo: bar;
  
  ul {
    foo: bar;
  }
 
  li {
    foo: bar;
  }
}

**Review by mpalmr — CHANGES_REQUESTED** [review]

File: src/components/index.jsx (hunk 4 above)
Comment (kagz) [reply src/components/index.jsx:53]: i just custom named it main-card sisnce its the one holding other elements

File: src/styles/index.scss (hunk 7 above)
Comment (kagz) [reply src/styles/index.scss]: thanks for this...i changed all the necessary places

File: src/styles/index.scss (hunk 5 above)
Comment (kagz) [reply src/styles/index.scss]: i did this ...thanks

File: src/components/index.jsx (hunk 2 above)
Comment (kagz) [reply src/components/index.jsx]: i removed it

File: src/components/index.jsx (hunk 4 above)
Comment (mpalmr) [reply src/components/index.jsx:53]: Why can't yo ustill use `<Container>` though?

**Review by mpalmr — APPROVED** [review]

**Pull request merged**